package.json

```diff
@@ -18,14 +18,18 @@
   "scripts": {
     "lint": "eslint .; sass-lint sass/* -v",
     "build": "webpack -p",
+    "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "webpack; http-server -p 3000 --cors -c-1"
   },
   "dependencies": {
-    "panzoom": "^2.5.0",
+    "panzoom": "^2.5.0"
+  },
+  "peerDependencies": {
```

Comment:
It is a bit controversial, I found that peerDependencies was kind of deprecated a long time ago https://medium.com/@dan_abramov/two-weird-tricks-that-fix-react-7cf9bbdef375 but everybody is still using them, e.g. `react-router` https://github.com/ReactTraining/react-router/blob/master/packages/react-router/package.json#L35